services/LibraryThing.filters.js

```diff
@@ -0,0 +1,38 @@
+export function selectOnlyTOS(document) {
+	const h2Titles = document.querySelectorAll('h2');
+	const titlePP = h2Titles[0];
+	const titleTOS = h2Titles[2];
```

Comment:
I think it's more robust to select section title by using text content, for example:
```js
const sectionTitles = Array.from(document.querySelectorAll('h2'));
const TOSTitle = sectionTitles.filter(el => el.textContent === "Terms of Service");
```